slackminion/plugins/test.py

```diff
@@ -31,12 +31,12 @@ def web_echo(self, foo):
         self.send_message(self.config['channel'], foo)
 
     @cmd()
-    def sleep(self, msg, args):
```

Comment:
This command name conflicted with the `sleep` command in `BasePlugin`, which caused the plugin loading to fail. Typically a user would not use the `TestPlugin` in a real environment, but for getting up and rolling this was the only plugin readily available for experimenting with the app.